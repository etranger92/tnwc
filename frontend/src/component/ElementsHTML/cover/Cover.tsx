import React from "react";
import IconHome from "../Icons/IconHome";
import "./cover.css";
//https://react-icons.github.io/react-icons/icons?name=fa

type Props = {
  loader: boolean;
};
const Cover: React.FC<Props> = (props) => {
  // eslint-disable-next-line react/prop-types
  const { loader } = props;
  return (
    <div id="cover-load" className={loader ? "hide_loader" : ""}>
      <IconHome />
    </div>
  );
};
export default Cover;
