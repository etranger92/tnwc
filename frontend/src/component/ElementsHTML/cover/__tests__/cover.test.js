import * as React from 'react';
import Cover from "../Cover.tsx"
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe("We test the innerHTML of the cover", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Cover />);
  });
  test("Snapshot cover", () => {
    //It works
 expect(toJson(wrapper)).toMatchSnapshot();
});
});
