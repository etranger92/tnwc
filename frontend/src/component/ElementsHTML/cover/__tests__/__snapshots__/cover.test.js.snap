// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`We test the innerHTML of the cover Snapshot cover 1`] = `
<div
  className=""
  id="cover-load"
>
  <IconHome />
</div>
`;
